app/elements/submission/tasks/general-code-task.html

```diff
@@ -164,10 +164,6 @@
 				this.control.loading();
 			},
 			finishTask: function(){
-				var msg = 'Are you sure to finish this task? You will not able to change come back again.';
-				if (!window.confirm(msg)) {
```

Comment:
i did not remove it. I moved it [here](https://github.com/coduno/app/pull/112/files#diff-198e20b282f93443c6266aa1a67f21ffR147)